### ISSUE TITLE
Starting and stopping timers doesn't work when there is a timer already running

### ISSUE THREAD

**Issue by tonnydourado** [toggl/track-extension#2149]
* Windows 10 Home, Windows 10 Enterprise N: 
* Firefox 108.0.2 (64-bit), Microsoft Edge 109.0.1518.52 (Official build) (64-bit), Microsoft Edge 108.0.1462.76 (Official build) (64-bit): 
* 2.0.18: 

#### Relevant integration (if any):

I tested with Gitlab (gitlab.com and an internal one that I don't know if I can mention public, so I won't =P) and Jira (atlassian.net)

### 🐛 Describe the bug

Starting or stopping timers, from both the button embedded in the page, or from the popup view, doesn't work if there is an already running timer: in the embedded button, the modal for specifying project, tags, etc. never shows up, and in the popup view, the timer just does not stop.

### Expected behaviour

Starting a timer from the embedded button or the extension popup should stop an existing timer

### Steps to reproduce

Steps to reproduce the behaviour:
1. Start a timer in the web apge
2. Go to an issue on any integration (tested with Gitlab and Atlassian.net Jira)
3. Try to start a new timer from the embedded button 

### Other details or context

In the console I can sometimes see an error about promises, but it doesn't appear always, and the message changes depending on the browser. On Firefox, it's something like `Error: Error: Promised response from onMessage listener went out of scope 4 [common.js:1:68501](moz-extension://2ecf6fe9-a95d-4f34-8dae-2165af70e2fd/scripts/common.js)`

If I stop the timer in the main web app, then things work as normal again.

**Comment by Nebucatnetzer** [comment]
I have the same problem since a few days.
I first thought it was broken in general but as @tonnydourado noted it only doesn't work when there is already a time working.

**Comment by andressltz** [comment]
I have the same problem on Chrome, started a few days ago.

**Comment by markoph** [comment]
I have same issue. I need to go to https://track.toggl.com to stop timer before starting next task.

- Fedora 37 (64-bit)
- Firefox 111.0.1 (64-bit)
- Extension version: 2.0.20
